controllers/front/ExpressCheckout.php

```diff
@@ -278,7 +278,7 @@ private function createAddress(
             $address->id_state = $idState;
         }
 
-        if ($address->validateFields(false)) {
+        if (true !== $address->validateFields(false)) {
```

Comment:
```suggestion
        if (!$address->validateFields(false)) {
```

Reply:
Address::validateFields can return bool or string

Reply:
@Matt75  in your case it will never return a string because `error_return` param is false. See `ObjectModel::validateFields` please

Reply:
Yes but we want strictly ensure case with bool